.coveragerc

```diff
@@ -0,0 +1,15 @@
+[run]
```

Comment:
As I mentioned on gitter, can we move this to a seperate PR?

Reply:
Sure thing, this is mostly developed while working on the feature, but I can split it off if you'd like.

Reply:
Opened https://github.com/iodide-project/iodide/pull/2305.